components/date-picker/date-popup.js

```diff
@@ -33,6 +33,7 @@ export default class DatePopup extends Component {
     time: PropTypes.string,
     from: dateType,
     to: dateType,
+    render: PropTypes.func,
```

Comment:
I like the idea in general but think naming could be more specific to be friendly for further extensions: say tomorrow we want to add custom content above the picker and we would have to name it like "renderBeforePicker". Then we would have "render" and "renderBeforePicker" and this naming seems confusing.

Maybe we would better rename "render" prop to "renderAfterPicker"?

Reply:
Agree, it's better to change naming.
What do you think about "renderAfterCalendar" ? Because when we open the date popup we see the date input, then weekdays, then the calendar with a list of months and years on its right. And then goes the place where visually I put a custom component.
Do you think "picker" wouldn't be confused with the date input ?

Reply:
Sure, this name is even better. Thanks!

Reply:
done